src/listeners/messageCreate.js

```diff
@@ -3,8 +3,8 @@ const CommandContext = require("../structures/CommandContext");
 module.exports = {
     name: "messageCreate",
     exec: async (client, msg) => {
-        if (!msg.guild) return;
-        if (msg.author.bot) return;     
+        if (!msg.guild || msg.author.bot || msg.webhookId) return;
```

Comment:
```suggestion
        if (!msg.guild || msg.author.bot) return;
```
webhooks is a bot too, i think it can be removed